### PR TITLE
Added Halloween Pumpkin filter in Snapchat filter

### PR DESCRIPTION
**Pull request by Khushan10**
# Related Issues or bug
  - Info about Issue or bug

Fixes: #[issue number that will be closed through this PR]

# Proposed Changes
 - Info about changes Goes here
 
# Additional Info
  - Anything related Issues 
  
# Screenshots

 Original           | Updated
 :--------------------: |:--------------------:
 **original screenshot** | **updated screenshot **|

### CODE REVIEW

**Review by akshitagupta15june — CHANGES_REQUESTED** [review]
can u add a screenshot of the output in readme

**Comment by akshitagupta15june** [comment]
@Khushan10

File: Snapchat_Filters/Halloween_Pumpkin/Readme.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Halloween Pumpkin Filter
+
```
Comment (Khushan10) [inline Snapchat_Filters/Halloween_Pumpkin/Readme.md:2]: Screenshot_20221101_034834

Comment (Khushan10) [reply Snapchat_Filters/Halloween_Pumpkin/Readme.md:2]: @akshitagupta15june  sorry for late response.

Comment (akshitagupta15june) [reply Snapchat_Filters/Halloween_Pumpkin/Readme.md:2]: remove the white background from picture, take a png pumpkin image

Comment (Khushan10) [reply Snapchat_Filters/Halloween_Pumpkin/Readme.md:2]: @akshitagupta15june  the image is png but i can't remove the white background, its my first open source contribution and working with face detection so little help would be appreciated.

**Comment by Khushan10** [comment]
@akshitagupta15june  Hey I have made some changes maybe something was wrong with old filter I have added new filter and a screenshot.

**Comment by Khushan10** [comment]
@akshitagupta15june  any update please??

**Comment by akshitagupta15june** [comment]
You can do one thing take that image of pumpkin and go to www.remove.bg and upload it and automatically your image background will be removed then download that image and give the path in ur code

**Comment by Khushan10** [comment]
> You can do one thing take that image of pumpkin and go to [www.remove.bg](http://www.remove.bg) and upload it and automatically your image background will be removed then download that image and give the path in ur code

I have used another image and now it works perfectly i have even attached the screenshot of the output of the working filter. Please check it @akshitagupta15june

**Review by akshitagupta15june — APPROVED** [review]
well done @Khushan10

**Pull request merged**